css/main.css

```diff
@@ -1,3 +1,225 @@
 /* Add your CSS here */
+*{
 
+}
+
+body{
+    /* display: grid; */
+    /* grid-template-columns: ; */
+    /* min-width: auto; */
+    margin-top: 5rem;
+    font-family: "poppins", sans serif;
+  }
+
+.topnav{
+    display: flex;
+    flex-direction: row;
+     
+}
+
+#myLinks a {
+  text-decoration: none;
+  color: black;
+  float: right;
+  width:7em;
+  margin:10px auto;
+  font-size: 15px;
+}
+
+
+.container{
+  width:100%;
+  display: grid;
+  grid-template-areas: " left right";
+  position:absolute;
+  top: 8rem;
+  padding: 120px 0 0 150px;
+  height: 32em;
+  right: -20em;
+  gap: px;
+}
+
+.main-text{
+    width: 50%;
+    grid-area: right;
+    margin: auto;
+    position: absolute;
```

Comment:
I think this is all a lot more complicated than it needs to be, and I think the place it's going wrong is the grey box. Try removing the grey box and all this absolute positioning. 

Have a look into CSS linear-gradient. Here's a codepen you might find interesting. https://codepen.io/stefanjudis/pen/vYOjJdZ?editors=1100